src/ds_caselaw_utils/courts.py

```diff
@@ -86,6 +86,32 @@ def get_selectable_groups(self):
                 groups.append(CourtGroup(category.get("display_name"), courts))
         return groups
 
+    def get_grouped_selectable_courts(self):
```

Comment:
Is it worth merging these into one core function which takes a filter lambda as a parameter like `lambda category: getattr(category, "is_tribunal")`?

Reply:
aha that's a nice idea, do you mind if i do a release then do that as a separate refinement in order to get the courts heirarchy out before i disappear for the rest of the week? For consistency we should also do the same with the listable courts methods, which would be a breaking change too.